test/apiv2/10-images.at

```diff
@@ -59,7 +59,7 @@ t GET  "images/testimage:20210610/json" 200 \
 
 # Make sure that new images are pulled
 old_iid=$(podman image inspect --format "{{.ID}}" docker.io/library/alpine:latest)
-podman rmi -f docker.io/library/alpine:latest
+11;rgb:0000/0000/0000podman rmi -f docker.io/library/alpine:latest
```

Comment:
copy-paste error, methinks. I also methinks that this will make tests barf before you get to the actual real test, so, you might want to repush.